src/js/pensions/components/FullNameField.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
 export default function FullNameField({ formData }) {
-  const { first, middle, last, suffix } = formData;
+  const { first, middle, last, suffix } = formData.fullName || formData;
```

Comment:
Ah, I added that when I was trying to use it on these pages, but I abandoned that idea. I will take this out.